x-pack/test/functional/services/cases/create.ts

```diff
@@ -58,18 +58,20 @@ export function CasesCreateViewServiceProvider({ getService, getPageObject }: Ft
       const descriptionArea = await find.byCssSelector('textarea.euiMarkdownEditorTextArea');
       await descriptionArea.focus();
       await descriptionArea.type(description);
-      await common.clickAndValidate(
-        'case-severity-selection',
-        `case-severity-selection-${severity}`
-      );
-      await testSubjects.click(`case-severity-selection-${severity}`);
+
+      if (severity !== CaseSeverity.LOW) {
+        await common.clickAndValidate(
+          'case-severity-selection',
+          `case-severity-selection-${severity}`
+        );
+      }
+
+      if (owner) {
+        await testSubjects.click(`${owner}RadioButton`);
+      }
 
       // save
       await testSubjects.click('create-case-submit');
-
-      await testSubjects.existOrFail('case-view-title', {
```

Comment:
The flyout does not redirect to the single case page. I make this helper function to work on both. I moved this check to the specific test that needs it.